### PR TITLE
Cult runes tweaks

### PR DESCRIPTION
**Pull request by Alberyk**
-you can't write runes on space or open space anymore
-teleporter runes do not work if they are on the central command level
-you can only summon nar'sie on the station's z levels
-improves the code a bit

### CODE REVIEW

File: code/game/gamemodes/cult/ritual.dm [hunk 1]
```diff
@@ -333,13 +332,25 @@ var/global/list/rnwords = list("ire","ego","nahlizet","certum","veri","jatkaa","
...
 		for(var/obj/effect/rune/N in rune_list)
 			C++
+
 		if (!istype(user.loc,/turf))
-			user << "<span class='warning'>You do not have enough space to write a proper rune.</span>"
+			to_chat(user, "<span class='warning'>You do not have enough space to write a proper rune.</span>")
+			return
+
+		var/turf/T = get_turf(user)
+
```
Comment (Arrow768) [inline code/game/gamemodes/cult/ritual.dm]: I would add the admin-z check here aswell

File: code/game/gamemodes/cult/runes.dm [hunk 2]
```diff
@@ -30,6 +30,12 @@ var/list/sacrificed = list()
 		if (istype(user, /mob/living))
 			user.take_overall_damage(5, 0)
 		qdel(src)
+
+	var/turf/T = get_turf(user)
+	if (T.z in current_map.admin_levels)
```
Comment (Arrow768) [inline code/game/gamemodes/cult/runes.dm]: I would move that to the general rune activation to not only limit the teleport rune, but all other runes aswell.

File: code/game/gamemodes/cult/ritual.dm (hunk 1 above)
Comment (Alberyk) [reply code/game/gamemodes/cult/ritual.dm]: My original plan was just to stop people from teleporting, you can still have someone write a rune in an admin z-level, like a merchant cultists sending a message or something.

File: code/game/gamemodes/cult/runes.dm (hunk 2 above)
Comment (Alberyk) [reply code/game/gamemodes/cult/runes.dm]: My original plan was just to stop people from teleporting, you can still have someone write a rune in an admin z-level, like a merchant cultists sending a message or something.

Comment (Arrow768) [reply code/game/gamemodes/cult/runes.dm]: Hm, I still feel that the ability to use runes on the merchant station should be completely removed.
(For the reasons explained in the topic)
If not, then the summon narnar rune should definitly be restricted.

Comment (Alberyk) [reply code/game/gamemodes/cult/runes.dm]: Alright, I will make so that you can only summon nar'sie on the station levels.

Comment (Alberyk) [reply code/game/gamemodes/cult/runes.dm]: Done, you now need to be on one of the station z-levels to summon nar'sie.

**Pull request merged**